README.md

```diff
@@ -51,3 +51,4 @@ module.exports = config;
 
 - **`aggregateFilename`**: The name of the file to be output that will get `.json` appended to it. Defaults to: `en-US`.
 
+- **`translatorFunction`**: (optional) Allows you to pass a translator function to translate the message, receives the final message text as a parameter
```

Comment:
should specify at least an example somewhere of how this is structured. for example, can I just pass in 
```
const myTranslateFunction = (input) => input.toUpperCase(); 
```

?